relayer/src/error.rs

```diff
@@ -30,6 +37,9 @@ pub enum Kind {
     #[error("RPC error to endpoint {0}")]
     Rpc(tendermint_rpc::Url),
 
+    #[error("call to Tendermint RPC returned error: {0}")]
+    TendermintRpcError(TendermintRpcError),
```

Comment:
Can we leave out the `Error` suffix in the variant names, here and elsewhere?

```suggestion
    TendermintRpc(TendermintRpcError),
```